### PR TITLE
rest: enrich request token email body

### LINKED ISSUE #273: tokens: more informative request token email body
When a user asks for token, we currently send the following email:

```
New user access token request:

id_: ....
email: john.doe@example.org
full_name: John Doe
username: johndoe
access_token: None
access_token_status: requested
```

It would be good to make it more user-friendly to REANA administrators, by doing:

(1) Add some greeting, perhaps:

```
Dear REANA administrator:
```

(2) Complete the message with actionable instructions, something like:

```
To accept the user token, please run:

$ kubectl ...

To reject the user token, please run:

$ kubectl ...
```

(3) Add some bottom line, perhaps simply:

```
Thanks,

<htttp://reana-qa.cern.ch>
```

### PR DESCRIPTION
**Pull request by mvidalgarcia**
closes #273

```
Dear REANA administrator,

New user access token request:

id_: 7df1b640-b9b4-41c8-ac6a-01923f1a8301
email: marco@reanahub.io
full_name: marco@reanahub.io
username: marco@reanahub.io
access_token: None
access_token_status: requested

To grant the user token, please run:

$ kubectl exec -ti deployment/reana-server -- flask reana-admin token-grant -e marco@reanahub.io --admin-access-token $REANA_ACCESS_TOKEN

To reject the user token, please run:

$ kubectl exec -ti deployment/reana-server -- flask reana-admin token-revoke -e marco@reanahub.io --admin-access-token $REANA_ACCESS_TOKEN

Thanks,

<reana.cern.ch>
```

### CODE REVIEW

File: templates/emails/token_request.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Dear REANA administrator,
+
+New user access token request:
```
Comment (diegodelemos) [inline templates/emails/token_request.txt:3]: What do you think about adding the `reana_url` in the beginning?
```suggestion
New user access token request in <{{ reana_url }}>:
```

Comment (tiborsimko) [reply templates/emails/token_request.txt:3]: Also, make it a phrase: 

```
There is a new access token request for http://reana-qa.cern.ch:
```

File: templates/emails/token_request.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+{{ user_data }}
+
+To grant the user token, please run:
+
+$ kubectl exec -ti reana-server-xxx -- flask reana-admin token-grant -e {{ user_email }} --admin-access-token $REANA_ACCESS_TOKEN
```
Comment (tiborsimko) [inline templates/emails/token_request.txt:9]: `kubectl exec -ti reana-server-xxx -- flask...` is not copy-paste friendly.

What about using `kubectl exec -i -t deployment/reana-server -- flask...` instead?

Comment (tiborsimko) [reply templates/emails/token_request.txt:9]: (BTW, we have `token-grant` but then `users-create`... Would be nice to harmonise *later* to use singular or plural consistently everywhere.)

**Review by diegodelemos — APPROVED** [review]

**Pull request merged**